models/music_cont_in_proxy.h

```diff
@@ -47,7 +47,7 @@
 namespace nest
 {
 
-/* BeginUserDocs: device, MUSIC
+/* BeginUserDocs: device, MUSIC, continuous
```

Comment:
```suggestion
/* BeginUserDocs: device, MUSIC
```

Another meaning to the max! This MUSIC proxy receives an array of floating point numbers per time step from somewhere else. The user can read them out of the status dict and do with them whatever she pleases to do. I don't really see a unified feature with the other things tagged `continuous` other than that they all operate on floats somehow.